docs/01_cva6_user/Programmer_View.rst

```diff
@@ -39,6 +39,7 @@ RISC-V Extensions
    "Zicount - Performance Counters",                                    "Yes","✓","✓"
    "Zicsr - Control and Status Register Instructions",                  "No","✓","✓"
    "Zifencei - Instruction-Fetch Fence",                                "No","✓","✓"
+   "Zicond - Integer Conditional Operations",                           "Yes","✓","✓"

```

Comment:
Can you reword "Zicond - Integer Conditional Operations (Ratification pending)"?